cmd/apprepository-controller/controller.go

```diff
@@ -564,6 +569,24 @@ func jobLabels(apprepo *apprepov1alpha1.AppRepository) map[string]string {
 	}
 }
 
+// ttlLifetimeJobs return time to live set by user if is nill return default
+func ttlLifetimeJobs(config Config) *int32 {
+	var result int32
+
+	if config.TTLSecondsAfterFinished == "" {
```

Comment:
Given the default command-line arg, a value of "" implies it's been explicitly overridden as "", in which case, I think we should not set the field? (as per the docs, if the field is not set, it won't be handled by the TTL controller).

To do this, you'd need to modify you earlier code only set the `TTLSecondsAfterFinished` field if `config.TTLSecondsAfterFinished != ""`. Thoughts?

Reply:
You are right absolutely